lib/system.nim

```diff
@@ -3756,6 +3756,13 @@ proc instantiationInfo*(index = -1, fullPaths = false): tuple[
   ##     testException(IndexError, tester(1))
   ##     # --> Test failure at example.nim:20 with 'tester(1)'
 
+proc lineInfoToString*(file: string, line, column: int): string =
+  when defined(lineInfoToStringSquashForTesting):
+    # useful for for unittests, see tfailedassert.nim
+    "FAKEFILE"
```

Comment:
This is just... please, just no.

Reply:
removed hardcoded FAKEFILE magic, see updated PR